WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/InsightsTabViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.stats.refresh
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.Transformations
+import kotlinx.coroutines.experimental.CoroutineScope
+import kotlinx.coroutines.experimental.launch
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.modules.UI_SCOPE
+import org.wordpress.android.ui.stats.refresh.InsightsUiState.StatsListState.DONE
+import javax.inject.Inject
+import javax.inject.Named
+
+class InsightsTabViewModel @Inject constructor(
+    @Named(UI_SCOPE) private val uiScope: CoroutineScope,
+    private val insightsViewModel: InsightsViewModel
+) : StatsListViewModel() {
+    private val _data = Transformations.map(insightsViewModel.data) { loadedData ->
+        InsightsUiState(loadedData, DONE)
```

Comment:
Not relevant to the current PR: We could do a little more magic with the UI state. For example on Refresh we'd reemit the UI state with refreshing state. I'm not sure it's necessary but I'll investigate it a bit more.